protocol/app/ante/replay_protection.go

```
@@ -86,7 +86,7 @@ func (rpd ReplayProtectionDecorator) AnteHandle(
 						1,
 						[]gometrics.Label{metrics.GetLabelForIntValue(metrics.ExecMode, int(ctx.ExecMode()))},
 					)
-					return ctx, errorsmod.Wrap(sdkerrors.ErrWrongSequence, err.Error())
+					return ctx, errorsmod.Wrapf(sdkerrors.ErrWrongSequence, err.Error())
```

golangci-lint (1.62.2)

printf: non-constant format string in call to cosmossdk.io/errors.Wrapf

(govet)